tests/__init__.py

```diff
@@ -105,7 +105,8 @@ def reset(self, delete_files = True):
         if delete_files:
             for file_name in os.listdir(self.directory):
                 file_path = os.path.join(self.directory, file_name)
-                if os.path.isfile(file_path):
+                if os.path.isfile(file_path) and (file_name not in ['log.txt']):
+                    logger.info('file path is %s' % file_path)
```

Comment:
Should this be config['log_file_name']

Reply:
nope, no such thing in testing... (this isn't the code)